pkg/health/condition/builder.go

```diff
@@ -23,6 +23,13 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+//
```

Comment:
do you want to add something like in comment for what `skipMergeConditions` is used